textile/features.textile

```diff
@@ -893,6 +915,27 @@ then the @enter@ request results in an error immediately.
 ** @(RTP15e)@ Implicitly attaches the @RealtimeChannel@ if the channel is in the @INITIALIZED@ state. However, if the channel is in or enters the @DETACHED@ or @FAILED@ state before the operation succeeds, it will result in an error
 ** @(RTP15f)@ If the client is identified and has a valid @clientId@, and the @clientId@ argument does not match the client's @clientId@, then it should indicate an error. The connection and channel remain available for further operations
 
+
+h3(#realtime-annotations). RealtimeAnnotations
+
+* @(RTAN1)@ @RealtimeAnnotations#publish@ function:
+** @(RTAN1a)@ Must accept the same arguments and performs the same validation, @Annotation.action@ setting, and data encoding as in @RSAN1@
+** @(RTAN1b)@ Has the same connection and channel state conditions as message publishing, see @RTL6c@
+** @(RTAN1c)@ Must put the annotation to be published into an array in the @annotations@ field of a @ProtocolMessage@ whose @action@ field is set to @ANNOTATION@, and @channel@ field is set to the channel name, and send it to the connection
+** @(RTAN1d)@ Must indicate success or failure of the publish (once @ACKed@ or @NACKed@) in the same way as @RealtimeChannel#publish@
+* @(RTAN2)@ @RealtimeAnnotations#delete@ function:
+** @(RTAN2a)@ Must be identical to @RTAN1@ @publish()@ except that the @Annotation.action@ is set to @ANNOTATION_DELETE@, not @ANNOTATION_CREATE@
+* @(RTAN3)@ @RealtimeAnnotations#get@ function:
+** @(RTAN3a)@ Is identical to @RestAnnotations#get@
+* @(RTAN4)@ @RealtimeAnnotations#subscribe@ function:
```

Comment:
Can we be explicit about what triggers a call to a listener here (the receipt of an `ANNOTATION` protocol message, presumably)?

Reply:
sure, clarified